charms/istio-pilot/tests/unit/test_charm.py

```diff
@@ -1601,6 +1601,9 @@ def test_use_https_with_tls_secret_not_found_with_id(
             in error_with_status.value.msg
         )
 
+    @pytest.mark.skip(
+        "Harness is not modeling the granted access correctly, see canonical/operator#1229"
+    )  # noqa
```

Comment:
Should we add a `FIXME` or `TODO`? I think that it's not clear that once the referenced issue is resolved, we expect to remove the `skip`

Reply:
I have added a `FIXME` in both the charm and the test file to make it more clear. Let me know what you think.